packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -113,7 +135,7 @@
     </dd>
     <dt>valign <code>enum</code></dt>
     <dd>
-      <p>If set, determines the vertical alignment of table cell (td) content. While the acceptable values contain all
+      <p>If set, determines the vertical alignment of table's cell (td) content. While the acceptable values contain all
```

Comment:
since we decided to leave all the alignment logic out of the first release, should we remove the `valign` argument handling too, in the component?

@Dhaulagiri what do you think?

Reply:
I think that makes sense. That'll give us time to also add showcase examples for how this alignment works in conjunction with the other ones we'll add.

Does removing it from the documentation feel sufficient? I'm not sure that the API for this would change based on the ideas we had for the other alignment features (this is applied to the entire table and not a column-by-column) basis.

Reply:
I would consider it a valuable part of the experiment to leave it and see what developers think when they have to use it, because it is similar to other properties that we decided to leave out. WDYT?

Reply:
> Does removing it from the documentation feel sufficient?

Yes, for now yes (it would be strange to state on the top of the page that alignment is still in progress, but expose this property in the component API).